image_geometry/include/image_geometry/pinhole_camera_model.h

```diff
@@ -270,7 +270,7 @@ class PinholeCameraModel
 
   // Use PIMPL here so we can change internals in patch updates if needed
   struct Cache;
-  boost::shared_ptr<Cache> cache_; // Holds cached data for internal use
+  std::shared_ptr<Cache> cache_; // Holds cached data for internal use
```

Comment:
that's actually an ABI breakage but I guess switching to C++11 is a higher priority for ROS2.